.github/workflows/sonar.yml

```diff
@@ -0,0 +1,30 @@
+name: SonarCloud analysis
+
+on: [push]
+
+jobs:
+  sonarcloud:
+    name: SonarCloud analysis
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '17' ]
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Can we use v4 version? Same applies for setup-java?

Reply:
yes, bumped to the latest version.

Reply:
yes, bumped to the latest version.